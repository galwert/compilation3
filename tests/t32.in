void printOK() {
    print("OK!");
    printi(5);
}

void main()
{
    int x = (((((5)))));
    byte Bro = 5 b; // really <Partner>?

    x = (((((x + Bro)+x)+x)));
    int Bro2 = (((((((((x+Bro))))))+1))+3);

    byte Bro3;
    Bro3 = (byte)x;
    x = (int)Bro3;
    x = (byte)Bro2;
    x = (bool)Bro2; // should fail here - cant cast int->bool
}