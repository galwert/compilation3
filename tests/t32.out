---end scope---
line 18: type mismatch
